src/config.rs

```diff
@@ -218,16 +219,45 @@ impl WriteOptions {
     }
 }
 
+/// Configuration for client put operations. `PutOptions` is supplied for each
+/// row inserted. This differs from [`WriteOptions`] in that a write may encompass
+/// multiple puts (such as the case with batched writes)
+#[derive(Clone, Default)]
+pub struct PutOptions {
+    /// The time-to-live (ttl) for this insertion. If this insert overwrites an existing
+    /// database entry, the TTL for the most recent entry will be canonical.
+    ///
+    /// Default: the TTL configured in DbOptions when opening a SlateDB session
```

Comment:
I think I missed this. Has this change been made?

Reply:
good catch - nope, not yet! I'll add that to the TODOs in the PR description (want to keep these as contained as possible, but happy to just have that in this PR if you prefer). For context, I copied the docs from the RFC doc 😅

Reply:
I decided to just go ahead and implement this in this PR since it lines up with some other changes i was doing